modules/gatsby/src/components/layout/header.jsx

```diff
@@ -126,6 +128,15 @@ export default class Header extends Component {
     };
   }
 
+  renderStars() {
+    const {config} = this.props;
+    if (config.PROJECT_TYPE === 'github') {
```

Comment:
Nit: We could either
- let the `<Github...>` react components know how to read the config (current design), and then we can move the `if (config.PROJECT_TYPE === 'github')` statement into the component.
- Alternative make the github components unaware of the config and destructure it and supply specific params before we call them.
- Mixing like this does not seem ideal.

Reply:
Maybe refactor later.